public/js/spec/apiSpec.js

```diff
@@ -0,0 +1,27 @@
+describe('Api', () => {
+  describe('fetchArticles', () => {
+    it('makes fetch request to given url', () => {
```

Comment:
not sure that the description is accurate since you're don't have an expectation around what URL the request is being made to